.golangci-ci.yaml

```diff
@@ -0,0 +1,67 @@
+# This is for the CI. To run it, please use:
+# golangci-lint run -c .golangci-ci.yaml.
+
+# Please keep the list in sync with .golangci-ide.yaml. For rationale, see the other file.
+
+linters:
+  # linters to run in addition to default ones
+  enable:
+    - dupl
+    - durationcheck
+    - errname
+    - errorlint
+    - exportloopref
+    - forbidigo
+    - forcetypeassert
+    - gci
+    - godot
+    - gofmt
+    - gosec
+    - ifshort
+    - misspell
+    - revive
+    - thelper
+    - tparallel
+    - unconvert
+    - unparam
+    - wastedassign
+    - whitespace
+    ##- wrapcheck # To think properly about it
+
+run:
+  # Most of the time, it’s taking less than 20s, but some builders are slower, so takes 2m
+  timeout: 2m
+
+# Get all linter issues, even if duplicated
+issues:
+  exclude-use-default: false
+  max-issues-per-linter: 0
+  max-same-issues: 0
+
+  exclude-rules:
+    # Prevents gci from complaining about the import "C" in our CGo files
+    - linters:
+      - gci
+      path: _c\.go
```

Comment:
After the `golangci` update, we don't need this section anymore. Now, in files where there is cgo, you can add a blank line between the `import "C"` and the actual imports and the linter is fine with it.
e.g
```Go
"random C code"
import "C"

import (
// random go imports
)
```